packages/ui/src/components/editor/properties/scene/settings/index.tsx

```diff
@@ -164,12 +171,39 @@ export const SceneSettingsEditor: EditorComponentType = (props) => {
     image.src = url
   }
 
+  const useSpectatingEntity = useState(sceneSettingsComponent.spectateEntity.value !== '')
+
   return (
     <PropertyGroup
       name={t('editor:properties.sceneSettings.name')}
       description={t('editor:properties.sceneSettings.description')}
       icon={<SceneSettingsEditor.iconComponent />}
     >
+      <InputGroup
+        name="Spectate Entity"
+        label={t('editor:properties.sceneSettings.lbl-spectate')}
+        info={t('editor:properties.sceneSettings.info-spectate')}
+      >
+        <BooleanInput value={useSpectatingEntity.value} onChange={useSpectatingEntity.set} />
+      </InputGroup>
+      {useSpectatingEntity.value ? (
+        <InputGroup
+          name="Entity UUID"
+          label={t('editor:properties.sceneSettings.lbl-uuid')}
+          info={t('editor:properties.sceneSettings.info-uuid')}
+        >
+          <SelectInput
```

Comment:
Can we use a NodeInput here instead to let people drag and drop entities instead of selecting from drop down?